pkgs/tools/system/lanzaboote/tool.nix

```diff
@@ -0,0 +1,50 @@
+{ systemd
+, makeWrapper
+, binutils-unwrapped
+, sbsigntool
+, rustPlatform
+, fetchFromGitHub
+, lib
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "lanzaboote-tool";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "nix-community";
+    repo = "lanzaboote";
+    rev = "v${version}";
+    hash = "sha256-Fb5TeRTdvUlo/5Yi2d+FC8a6KoRLk2h1VE0/peMhWPs=";
+  };
+
+  sourceRoot = "source/rust/tool";
+  cargoHash = "sha256-ebaLnq2takCEvbHVqkI73FVTwjn1u80nnuy+GzVnROA=";
+
+  env.TEST_SYSTEMD = systemd;
```

Comment:
Ideally, we could make this dependency conditional on `doCheck`, but we don't have `finalAttrs` support for `buildRustPackage` yet #194475. We could have an additional `doCheck` argument but I have no strong opinion.

Reply:
Maybe we should just use `lib.meta.availableOn` ? Follow up PRs are welcome on the matter.

Reply:
Addressed.